R/analyze_compile_one.R

```diff
@@ -23,6 +23,11 @@ deepstate_fuzz_fun<-function(fun_path,seed=-1,time.limit.seconds=-1){
   test_harness.o <- file.path(fun_path, paste0(fun_name, "_DeepState_TestHarness.o"))
   test_harness_path <- file.path(fun_path,paste0(fun_name,"_DeepState_TestHarness"))
   test_harness <- paste0(fun_name,"_DeepState_TestHarness")
+  ## If time.limit.seconds is lessthan or equal to zero we return NULL
+  if(time.limit.seconds <= 0){
+    message(sprintf("time.limit.seconds should always be greater than zero"))
```

Comment:
stop instead of message(sprintf
also please don't use sprintf if there are no format strings (%s etc) to convert, it is confusing

Reply:
Using stop(). Removed sprintf when no %s is used.